testing/util/altair.go

```diff
@@ -377,6 +377,12 @@ func GenerateFullBlockAltair(
 		}
 	}
 
+	syncCommitteeBits := make(bitfield.Bitvector512, 64)
```

Comment:
This function prepared a full SyncAggregate, but the signature would fail at the fork. Fixing that perhaps is probable, but since this is not used anywhere on any test, I decided that it was easier to just return the empty participation case.

Reply:
See below: I had to reuse the previous behavior because of the monitor's test